addons/mouse-pos/stage-size.js

```diff
@@ -0,0 +1,69 @@
+export const stageEventTarget = new EventTarget();
```

Comment:
Is there a reason this module is in `mouse-pos`?

Reply:
I think it's the first addon that added its own UI to the top of the stage.